File: Exercism/users/jjdub7/python/raindrops/raindrops.py
def convert(number):

    sound = ""

    if int(number) % 3 == 0:
        sound = sound + 'Pling'
    if int(number) % 5 == 0:
        sound = sound + 'Plang'
    if int(number) % 7 == 0:
        sound = sound + 'Plong'
    if int(number) % 3 * int(number) % 5 * int(number) % 7 > 0:
        sound = str(number)

    return sound

    pass

File: Exercism/cpp/raindrops/raindrops.cpp
#include "raindrops.h"

#include <string>

using namespace std;

namespace raindrops {

	string convert(int i) {
		string ret = "";

		if (i % 3 == 0) ret += "Pling";
		if (i % 5 == 0) ret += "Plang";
		if (i % 7 == 0) ret += "Plong";

		if (ret == "") {
			return to_string(i);
		}
		else {
			return ret;
		}
	}

}  // namespace raindrops
